### DISCUSSION TITLE
Tubecast

### DISCUSSION THREAD

**Discussion by petanque** [music-assistant/support#1613] [Feature requests and ideas]
I use tubecast for kodi, with that I can cast from the youtube app on android.
Could it be an feature to add or what do you think?
https://github.com/enen92/script.tubecast

**Comment by OzGav** [comment]
As stated in that repo it will only work while Google supports Cast V1. We already have regular problems maintaining the YouTube music provider as it has no official API. Pretty much the recommendation is for people to move to another music provider that has an API so they aren't inconvenienced with constant breakages. Someone would have to develop this as well and there is no spare capacity at the moment.

**Comment by OzGav** [comment]
Doing a sweep and closing old outdated posts
